### ISSUE TITLE
Does conduit has any plans to integrate with SPIFFE(https://spiffe.io/)?

### ISSUE THREAD

**Issue by zekefast** [runconduit/conduit#768]

**Comment by olix0r** [comment]
@zekefast Thanks for asking! Spiffe is definitely an interesting project that we've been following since its inception as a whitepaper.

Conduit's going to start with a simpler form of identity management, so that we can help start bootstrapping identity without requiring external configuration. Once the fundamental building blocks of identity (namely, mutual TLS) are established in Conduit, we plan to have a good integration path with other identity providers like Spiffe.

**Comment by erwbgy** [comment]
Now that MTLS is available in Linkerd2, is SPIFFE support going to be revisited?

**Comment by wmorgan** [comment]
@erwbgy it's definitely on our minds. Would love to find a concrete user who has a clear use case for both. Know any? :)

**Comment by erwbgy** [comment]
AIUI one of the key featues of SMI is to apply policies like identity across services and I think users would get much better interoperability if this identity was based on a neutral, common spec like SPIFFE.  

This could also provide a much easier migration path from an existing more complex service mesh that uses SPIFFE to Linkerd2.

**Comment by wmorgan** [comment]
Agreed. My point was more, finding a concrete user with a clear use case for Linkerd + SPIFFE will help us prioritize this effort.
